.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,12 +2,14 @@
 documentation before submitting a Pull Request!
 Thank you for contributing to Rook! -->
 
-Description of your changes:
+**Description of your changes:**
 
-Which issue is resolved by this Pull Request:
+**Which issue is resolved by this Pull Request:**
 Resolves #
 
-Checklist:
+**Checklist:**
 - [ ] Documentation has been updated, if necessary.
 - [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
+- [ ] Code generation has been run to update object specifications.
```

Comment:
include the command here? `make codegen`

Reply:
@jbw976 Done.